server/odc-core/src/main/java/com/oceanbase/odc/core/sql/execute/mapper/DefaultJdbcRowMapper.java

```diff
@@ -46,6 +46,7 @@ public DefaultJdbcRowMapper(@NonNull ConnectionSession session) {
             mapperList.add(new MySQLDatetimeMapper());
             mapperList.add(new MySQLYearMapper());
             mapperList.add(new MySQLTimestampMapper());
+            mapperList.add(new MySQLGeometryMapper());
```

Comment:
should it be used on rollback plan?

Reply:
Yes. add MySQLGeometryMapper in RollBackPlanJdbcRowMapper